<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sugar</class>
 <widget class="QWidget" name="Sugar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblLastRec">
     <property name="text">
      <string>Last Record</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="tblViewRec"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lblComment">
     <property name="text">
      <string>Comment for new record:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="txtComment">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblInsert">
     <property name="text">
      <string>Insert Rec</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblSugar">
     <property name="text">
      <string>Sugar</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="btnInsert">
     <property name="text">
      <string>A&amp;dd</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>E&amp;xit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="sugarCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
